src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -50,33 +50,34 @@ import UpcomingTickets from "components/molecules/UpcomingTickets";
 import "./NavBar.scss";
 import "./playa.scss";
 
-const TicketsPopover: React.FC<{ futureUpcoming: UpcomingEvent[] }> = ({
-  futureUpcoming,
-}) => (
-  <Popover id="popover-basic">
+const TicketsPopover: React.FC<{ futureUpcoming: UpcomingEvent[] }> = (
+  props: unknown,
+  { futureUpcoming }
+) => (
+  <Popover id="popover-basic" {...props}>
     <Popover.Content>
       <UpcomingTickets events={futureUpcoming} />
     </Popover.Content>
   </Popover>
 );
 
-const ChatPopover: React.FC = () => (
+const ChatPopover = (
   <Popover id="popover-basic">
     <Popover.Content>
       <PrivateChatModal />
     </Popover.Content>
   </Popover>
 );
 
-const ProfilePopover: React.FC = () => (
+const ProfilePopover = (
   <Popover id="profile-popover">
     <Popover.Content>
       <ProfilePopoverContent />
     </Popover.Content>
   </Popover>
 );
 
-const GiftPopover: React.FC = () => (
+const GiftPopover = (
```

Comment:
I wouldn't have changed these. As they were they were react subcomponents.

What was the thoughts behind changing them to this style?